sei-devnet-2/genesis.json

```diff
@@ -0,0 +1,1475 @@
+            },
+            {
+                "date": "2023-02-17",
+                "token_release_amount": "100000000"
+            },
+            {
+                "date": "2023-02-24",
+                "token_release_amount": "100000000"
+            },
+            {
+                "date": "2023-03-03",
+                "token_release_amount": "100000000"
+            },
+            {
+                "date": "2023-03-10",
+                "token_release_amount": "100000000"
+            },
+            {
+                "date": "2023-03-17",
+                "token_release_amount": "100000000"
+            },
+            {
+                "date": "2023-03-24",
+                "token_release_amount": "100000000"
+            },
+            {
+                "date": "2023-03-31",
+                "token_release_amount": "100000000"
+            },
+            {
+                "date": "2023-04-07",
+                "token_release_amount": "100000000"
+            },
+            {
+                "date": "2023-04-14",
+                "token_release_amount": "100000000"
+            },
+            {
+                "date": "2023-04-21",
+                "token_release_amount": "100000000"
+            },
+            {
+                "date": "2023-04-28",
+                "token_release_amount": "100000000"
+            },
+            {
+                "date": "2023-05-05",
+                "token_release_amount": "100000000"
+            },
+            {
+                "date": "2023-05-12",
+                "token_release_amount": "100000000"
+            },
+            {
+                "date": "2023-05-19",
+                "token_release_amount": "100000000"
+            },
+            {
+                "date": "2023-05-26",
+                "token_release_amount": "100000000"
+            },
+            {
+                "date": "2024-06-01",
+                "token_release_amount": "1000000000"
+            },
+            {
+                "date": "2025-06-01",
+                "token_release_amount": "1000000000"
+            },
+            {
+                "date": "2026-06-01",
+                "token_release_amount": "1000000000"
+            },
+            {
+                "date": "2027-06-01",
+                "token_release_amount": "1000000000"
+            },
+            {
+                "date": "2028-06-01",
+                "token_release_amount": "1000000000"
+            },
+            {
+                "date": "2029-06-01",
+                "token_release_amount": "1000000000"
+            }
+        ]
+      }
+    },
+    "nitro": {
+      "params": {
+        "whitelisted_tx_senders": []
+      },
+      "slot": "0",
+      "sender": "",
+      "stateRoot": "",
+      "txs": []
+    },
+    "oracle": {
+      "params": {
+        "vote_period": "10",
+        "vote_threshold": "0.500000000000000000",
+        "reward_band": "0.020000000000000000",
```

Comment:
@udpatil PTAL and confirm

Reply:
we want vote_period to be `1`

do we expect other validators to be running oracle processes for dev-2 or not?

if not, we should lower the `vote_threshold` to `0` since otherwise we won't be able to get oracle prices finalized if we're the only ones running oracle processes. For atlantic-2 we definitely want this oracle `vote_threshold` to be with `0.5` or `0.67` depending on how much voting power we determine is necessary for updating oracle prices, but it may not make since to have such a high threshold for devnet. We probably do want at some point in devnet-2 to have validators ALL run oracle processes to identify any issues with the vote period of 1, and at that time we could plan to raise the vote_threshold via governance to test a similar environment as atlantic-2 or eventual mainnet.

@philipsu522 thoughts?

Reply:
The main reason for this testing period for devnet-2 would be to verify that we can properly come to consensus on oracle prices at scale with voting period of 1 in a more real world environment as opposed to a highly performant loadtest testnet cluster

Reply:
as discussed with @udpatil , we set this to 0, and we will ask validators to turn this on later